src/labels/CustomLabels.labels

```diff
@@ -2127,6 +2135,38 @@
         <shortDescription>Plural Picklist substitute for Yearly</shortDescription>
         <value>Years</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentMessage</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Spinner text when Elevate Commitment is being created</shortDescription>
+        <value>Creating Elevate commitment... This might take some time, please wait.</value>
```

Comment:
```suggestion
        <value>Creating Elevate commitment...This might take some time, please wait.</value>
```

Reply:
@RadSFDeveloper - Do we have a sense of how much time a person might need to wait for the commitment to be created? We probably don't need the "please wait" if it's only a matter of seconds.

Reply:
Max two min, but usually it is few seconds.